snmp/datadog_checks/snmp/data/profiles/palo-alto.yaml

```diff
@@ -63,4 +63,56 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.25461.2.1.2.5.1.3
       name: panGPGWUtilizationActiveTunnels
-
+  - MIB: PAN-ENTITY-EXT-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.25461.1.1.7.1.1.2
+      name: panEntityTotalPowerUsed
+    # metric_tags:
+    #   - MIB: ENTITY-MIB
+    #     column:
+    #       OID: 1.3.6.1.2.1.47.1.2.1.1.2
+    #       name: entLogicalDescr
+    #     table: entLogicalTable
+    #     tag: ent_descr
```

Comment:
Is tagging a symbol (non-table metric) with a table specific OID possible?

Reply:
No that doesn't make sense. Let's remove this.